src/models/user.ts

```diff
@@ -72,10 +73,16 @@ const UserSchema = new mongoose.Schema(
   },
 );
 
-UserSchema.virtual('event', {
-  ref: 'Event',
+UserSchema.virtual('position', {
+  ref: ModelRefs.POSITION,
   localField: '_id',
-  foreignField: 'owner',
+  foreignField: 'occupant',
+});
+
+UserSchema.virtual('nomination', {
+  ref: ModelRefs.NOMINATION,
+  localField: '_id',
+  foreignField: 'candidate',
```

Comment:
Maybe this should be plural too?

Reply:
I don't think so. Every nomination can have 1 candidate. If you want to nominate more people for the same position you'd have to create new nominations.

Reply:
I'm pretty sure here you are defining a virtual `nomination` property on `User` docs that is an array of references to `Nomination` docs. It will include all `Nomination` docs with `candidate` field matching the `User` doc's `_id`.

Reply:
updated in 775aec3f4620d07c84cc9b51685b62bcfd93f732.